Controller/Adminhtml/Account/Open.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Copyright (c) 2020, Nosto Solutions Ltd
+ * Copyright (c) 2022, Nosto Solutions Ltd
```

Comment:
Why was this file permission changed to 644? Please revert to avoid issues

Reply:
File permissions in the test instance? I have not deployed my changes anywhere yet.

Reply:
I don't know why, maybe you created a new file, but the permissions are wrong:
![image](https://user-images.githubusercontent.com/2778820/182107914-7d0ea787-232e-4222-a4e4-baf66e2c8ab9.png)